packages/peregrine/lib/hooks/usePagination.js

```diff
@@ -43,12 +43,12 @@ export const usePagination = ({
     initialPage,
     initialTotalPages = 1
 } = {}) => {
+    const searchParam = namespace ? `${namespace}_${parameter}` : parameter;
```

Comment:
This only includes  an `_` if namespace is provided. It was incorrectly using a param of `_page`.